### PR TITLE
NH3 + CO DFRobot Multigas

### PR DESCRIPTION
**Pull request by roberbike**
First step to add DFRobot Multigas sensor. 
Many errors, in this first step. could you guide me?

### CODE REVIEW

File: src/Sensors.hpp [hunk 1]
```diff
@@ -17,9 +17,10 @@
 #include <s8_uart.h>
 #include <sps30.h>
 #include <drivers/pm1006.h>
+#include <drivers/DFRobot_MultiGasSensor.h>
 
-#define CSL_VERSION "0.5.9"
-#define CSL_REVISION 366
+#define CSL_VERSION "0.5.9.1"
```
Comment (hpsaturn) [inline src/Sensors.hpp:22]: this is invalid, only add one more to the final number of revision and version.

More info here:
https://docs.platformio.org/en/latest/manifests/library-json/fields/version.html

File: src/Sensors.hpp [hunk 2]
```diff
@@ -251,21 +259,22 @@ class Sensors {
 
     float getAltitude();
 
-    float getGas();
+    // float getGas();    // Posible colision con DFRobot Multigas
```
Comment (hpsaturn) [inline src/Sensors.hpp:262]: I understand you. Maybe could be remove the gas method, or change it to real unit, a specific unit. Like, getCO2, getCO, getPM25.

File: src/drivers/DFRobot_MultiGasSensor.h [hunk 3]
```diff
@@ -0,0 +1,326 @@
+/*!
```
Comment (hpsaturn) [inline src/drivers/DFRobot_MultiGasSensor.h:1]: We don't need this file here. It is already included in the library dependency. Please remove it.

File: src/drivers/DFRobot_MultiGasSensor.cpp [hunk 4]
```diff
@@ -0,0 +1,95 @@
+/*!
```
Comment (hpsaturn) [inline src/drivers/DFRobot_MultiGasSensor.cpp:1]: We don't need this file here. It is already included in the library dependency. Please remove it.

**Review by hpsaturn — CHANGES_REQUESTED** [review]
Thanks @roberbike for your contribution. Please only do the changes of my comments. I could finish the implementation after that. Don't worries :)

**Comment by roberbike** [comment]
Done.

File: examples/gravity/readGasConcentration.cpp [hunk 5]
```diff
@@ -0,0 +1,95 @@
+/*!
```
[the remark this answers is not in the document]
Comment (roberbike) [reply examples/gravity/readGasConcentration.cpp]: Eliminado

File: src/Sensors.cpp [hunk 6]
```diff
@@ -557,8 +559,12 @@ float Sensors::getUnitValue(UNIT unit) {
             return pres;
         case ALT:
             return alt;
-        case GAS:
-            return gas;
+       // case GAS:
```
[the remark this answers is not in the document]
Comment (roberbike) [reply src/Sensors.cpp:562]: EL bmp680 utiliza el mismo nombre de variable, gas. Una de ellas deberi de cambiar.

File: src/Sensors.hpp (hunk 1 above)
Comment (roberbike) [reply src/Sensors.hpp:22]: Entendido

File: src/Sensors.hpp (hunk 2 above)
Comment (roberbike) [reply src/Sensors.hpp:262]: // float getGas();    // Posible colision con DFRobot Multigas
    
    float getNH3();
    
    float getCO();

File: src/drivers/DFRobot_MultiGasSensor.cpp (hunk 4 above)
Comment (roberbike) [reply src/drivers/DFRobot_MultiGasSensor.cpp]: eliminado

File: src/drivers/DFRobot_MultiGasSensor.h (hunk 3 above)
Comment (roberbike) [reply src/drivers/DFRobot_MultiGasSensor.h]: Eliminado

File: src/Sensors.hpp (hunk 1 above)
Comment (hpsaturn) [reply src/Sensors.hpp:22]: Bueno, aqui deberia quedar 0.5.10 si fue un cambio pequeño compatible. O 0.6.0 si no es compatible hacia atras. Q creo es el caso por el tema del gas, porque ese getter depronto lo vamos a cambiar

File: src/Sensors.hpp [hunk 7]
```diff
@@ -17,9 +17,10 @@
 #include <s8_uart.h>
 #include <sps30.h>
 #include <drivers/pm1006.h>
+#include <drivers/DFRobot_MultiGasSensor.h>
 
-#define CSL_VERSION "0.5.9"
-#define CSL_REVISION 366
+#define CSL_VERSION "0.5.9.1"
+#define CSL_REVISION 366.1
```
Comment (hpsaturn) [inline src/Sensors.hpp:23]: aqui simplemente sube el numero.

**Review by hpsaturn — APPROVED** [review]

**Pull request merged**